AnaMamani/ContactManager/src/com/company/Console.java

```diff
@@ -0,0 +1,89 @@
+package com.company;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class Console {
+
+    Scanner sn;
+    boolean out;
+    ContactManager contacts;
+    int option;
+
+    public Console() {
+        this.sn = new Scanner(System.in);
+        this.out = false;
+        this.contacts = new ContactManager();
+    }
+
+    public void ContactMenu() {
```

Comment:
Method name always has to start with a verb because a method is an action of the class.